### PR TITLE
Op with resource

### PR DESCRIPTION
**Pull request by applenob**
# Pull Request Summary
Update customer op `SentenceToIds` and `JiebaCut`. These two op now accept `list(string)` as attributes which can be added into a `.pbtxt` file. Thus we don't need extra resource files when deploying.

# Test Plan
tested.

### CODE REVIEW

File: delta/data/preprocess/text_ops.py [hunk 1]
```diff
@@ -17,17 +17,21 @@
 
 import tensorflow as tf
 from absl import logging
+import os
 
 from delta import utils
 from delta.layers.ops import py_x_ops
+from delta.data.utils import read_lines_from_text_file
```
Comment (zh794390558) [inline delta/data/preprocess/text_ops.py]: make `read_lines_from_text_file` a local func.

File: delta/layers/ops/kernels/jieba_op.cc [hunk 2]
```diff
@@ -36,19 +36,39 @@ namespace delta {
...
+      OP_REQUIRES_OK(ctx, ctx->GetAttr("dict_lines", &dict_lines));
+      std::vector<std::string> model_lines;
+      OP_REQUIRES_OK(ctx, ctx->GetAttr("model_lines", &model_lines));
+      std::vector<std::string> user_dict_lines;
+      OP_REQUIRES_OK(ctx, ctx->GetAttr("user_dict_lines", &user_dict_lines));
+      std::vector<std::string> idf_lines;
+      OP_REQUIRES_OK(ctx, ctx->GetAttr("idf_lines", &idf_lines));
+      std::vector<std::string> stop_word_lines;
+      OP_REQUIRES_OK(ctx, ctx->GetAttr("stop_word_lines", &stop_word_lines));
+      OP_REQUIRES_OK(ctx, ctx->GetAttr("hmm", &hmm));
```
Comment (zh794390558) [inline delta/layers/ops/kernels/jieba_op.cc]: `hmm` not used?

File: delta/data/preprocess/text_ops.py (hunk 1 above)
Comment (applenob) [reply delta/data/preprocess/text_ops.py]: It was also used in `jieba_op_test.py`.

File: delta/layers/ops/kernels/jieba_op.cc (hunk 2 above)
Comment (applenob) [reply delta/layers/ops/kernels/jieba_op.cc]: fixed

**Pull request merged**